lib/discourse_chat/provider/slack/slack_provider.rb

```diff
@@ -56,6 +71,11 @@ def self.slack_message(post, channel)
       attachments: []
     }
 
+    if filter == "thread"
+      thread_ts = topic.get_slack_thread_id
+      message[:thread_ts] = thread_ts if not thread_ts.nil?
+    end
+
```

Comment:
```suggestion
    if filter == "thread" && thread_ts = topic.slack_thread_id
      message[:thread_ts] = thread_ts
    end

```

Reply:
I keep forgetting about `=` in `if` being OK in Ruby.